omnigibson/envs/env_base.py

```diff
@@ -280,6 +268,33 @@ def _load_objects(self):
 
         assert og.sim.is_stopped(), "Simulator must be stopped after loading objects!"
 
+    def _load_external_sensors(self):
+        """
+        Load any additional custom external sensors into the scene
+        """
+        assert og.sim.is_stopped(), "Simulator must be stopped before loading external sensors!"
+        sensors_config = self.env_config["external_sensors"]
+        if sensors_config is not None:
+            self._external_sensors = dict()
+            for i, sensor_config in enumerate(sensors_config):
+                # Add a name for the object if necessary
+                if "name" not in sensor_config:
+                    sensor_config["name"] = f"external_sensor{i}"
```

Comment:
maybe an extra _ after sensor?

Reply:
bit of a nit, keeping this consistent with our other naming convention (e.g.: `robot0` vs `robot_0`)